### DISCUSSION TITLE
A few housekeeping questions!

### DISCUSSION THREAD

**Discussion by andreasrf** [ua-gist604b-f24/syllabus#20] [General]
I'm looking to get started on Module 6! Is there a timeline on when Modules 6-7 will be available? Additionally, when can we expect an update on our grades? It's been a few modules and I'm a little nervous that I've missed something. 

Thank you!
Andrea

**Comment by aldunn4** [comment]
I was wondering the same, I only have grades for Modules 0 and 1.
